playbooks/cluster-up.yaml

```diff
@@ -1,13 +1,20 @@
-# Passable Vars ('-e var=val')
+# Passable Vars ('-e var1=val1 var2=val2')
```

Comment:
are you sure the quotes surround the `-e` too? Just seems a bit odd to me...

Reply:
woops - right, they don't.  will fix in a one-liner